### ISSUE TITLE
Camera: let the user focus the target before capturing

### ISSUE THREAD

**Issue by scit2010** [osmandapp/Osmand#3159]
System: 
Samsung Galaxy S4, Osmand+ 2.4.7, Android Cyanogenmod 12.1

How to reproduce:
1. Activate the audio/video note plugin.
2. At the current position tap the screen to take a photo.

Expected result:
Osmand+ starts the internal or external camera app to take a photo. It doesn't capture the photo automatically, but lets the user focus the target first and capture the photo when it is in focus. It takes only 1 (one) attempt to capture the desired photo in the right way.

Actual result:
Osmand+ starts the camera and captures immediately. 
I often have to make two or more attempts to take a proper photo with Osmand+ which is very annoying.
Remember that the user often holds the phone in his/her hands and the phone camera is directed down to the floor. So the first image is almost everytime a photo of the ground or doesn't focus the target entirely. Please fix this.

**Comment by polarbearing** [comment]
Cannot reproduce on my 2.4.1 device. The button asks which kind of note to take, asks which camera app to use, and the stock camera app shows me a blue trigger button. After capturing I can confirm or discard the image. Tried a different camera app, same result.

**Comment by scit2010** [comment]
You're right for the case of using the system camera. I tried again and faced following effects:
If using the system camera: it works as described in your comment #1. Formerly the behavior of my bug report occurred here, too. Seems to be an occasional effect.
If using Osmand+'s internal camera: same as described in my bug report. I tried the little round button in the bottom left corner. It lets me make another photo until the ring around the check-mark button in the middle completed, so it seems to meet my need. But at my third attempt the system camera app opens immediately. This is a weird and unexpected effect - is it by intention?
